e2e/messages/addIPFSMessage.test.ts

```diff
@@ -5,7 +5,7 @@ import { base32 } from 'multiformats/bases/base32';
 import { CID } from 'multiformats/cid';
 import { PARQUET_BROADCAST } from '../schemas/fixtures/parquetBroadcastSchemaType';
 import assert from 'assert';
-import { assertHasMessage, createAndFundKeypair } from '../scaffolding/helpers';
+import { assertHasMessage, createAndFundKeypair, getOrCreateDummySchema } from '../scaffolding/helpers';
```

Comment:
See helpers for more info, but `getOrCreateSchemaV3` does just that! It checks the schema name, and creates one only if needed.